### PR TITLE
This translates the options from task-name to something a user will understand.

### PR DESCRIPTION
**Pull request by Zedb0T**
Currently Hub 1 is done.

### CODE REVIEW

File: jak1/opengoal-jak1-autosplitter.asl [hunk 1]
```diff
@@ -39,199 +39,201 @@ startup {
...
+  AddOption(vars.swampResolutions, "res_swamp_flutflut", structByteIdx++, typeof(byte), 1, false, "Ride the Flut Flut", false);
+  AddOption(vars.swampResolutions, "res_swamp_battle", structByteIdx++, typeof(byte), 1, false, "Defeat the Lurker Ambush", false);
+  AddOption(vars.swampResolutions, "res_swamp_tether_1", structByteIdx++, typeof(byte), 1, false, "Break the first tether to the Zeppelin", false);
+  AddOption(vars.swampResolutions, "res_swamp_tether_2", structByteIdx++, typeof(byte), 1, false, "Break the second tether to the Zeppelin", false);
+  AddOption(vars.swampResolutions, "res_swamp_tether_3", structByteIdx++, typeof(byte), 1, false, "Break the third tether to the Zeppelin", false);
+  AddOption(vars.swampResolutions, "res_swamp_tether_4", structByteIdx++, typeof(byte), 1, false, "Break the fourth tether to the Zeppelin", false);
+  AddOption(vars.swampResolutions, "res_swamp_buzzer", structByteIdx++, typeof(byte), 1, false, "Free 7 Scout Flies", false);
+  //While this is a "need res task" I think its more clear if we move it to a cutscenes category and rename this category "Power cells" Or something
```
Comment (Zedb0T) [inline jak1/opengoal-jak1-autosplitter.asl:128]: //While this is a "need res task" I think its more clear if we move it to a cutscenes category and rename this category "Power cells" Or something

File: jak1/opengoal-jak1-autosplitter.asl [hunk 2]
```diff
@@ -39,199 +39,201 @@ startup {
...
+  AddOption(vars.snowyResolutions, "res_snow_eggtop", structByteIdx++, typeof(byte), 1, false, "Find the Yellow Vent switch", false);
+  AddOption(vars.snowyResolutions, "res_snow_ram", structByteIdx++, typeof(byte), 1, false, "Stop the 3 Lurker Glacier Troops", false);
+  AddOption(vars.snowyResolutions, "res_snow_fort", structByteIdx++, typeof(byte), 1, false, "Get through the Lurker Fort", false);
+  AddOption(vars.snowyResolutions, "res_snow_ball", structByteIdx++, typeof(byte), 1, false, "Open the Lurker Fort Gate", false);
+  AddOption(vars.snowyResolutions, "res_snow_bunnies", structByteIdx++, typeof(byte), 1, false, "Survive the Lurker Infested Cave", false);
+  AddOption(vars.snowyResolutions, "res_snow_buzzer", structByteIdx++, typeof(byte), 1, false, "Free 7 Scout Flies", false);
+  AddOption(vars.snowyResolutions, "res_snow_bumpers", structByteIdx++, typeof(byte), 1, false, "Deactivate the Precursor Blockers", false);
+  AddOption(vars.snowyResolutions, "res_snow_cage", structByteIdx++, typeof(byte), 1, false, "Opent the Frozen Crate", false);
+  //The task below is unsed in retail versions of the game.
```
Comment (Zedb0T) [inline jak1/opengoal-jak1-autosplitter.asl:172]: //The task below is unsed in retail versions of the game.

**Comment by Zedb0T** [comment]
Also, none of the misc tasks seemed to work for me but I still need to investigate if I messed up or something.

When are they expected to trigger?
```  AddOption(vars.miscallenousTasks, "int_finalboss_movies", structByteIdx++, typeof(byte), 1, true, "int_finalboss_movies", false);
  AddOption(vars.miscallenousTasks, "unk_finalboss_movies", structByteIdx++, typeof(byte), 1, false, "unk_finalboss_movies", false);
  AddOption(vars.miscallenousTasks, "int_jungle_fishgame", structByteIdx++, typeof(byte), 1, false, "int_jungle_fishgame", false);```

**Review by xTVaser — APPROVED** [review]
looks good, just changing the visible label

**Pull request merged**